packages/store/internals/src/index.ts

```diff
@@ -1,12 +1,9 @@
-export { NgxsBootstrapper } from './ngxs-bootstrapper';
-export { memoize } from './memoize';
-export { INITIAL_STATE_TOKEN, InitialState } from './initial-state';
-export {
-  PlainObjectOf,
-  PlainObject,
-  ɵStateClass,
-  ɵMETA_KEY,
-  ɵMETA_OPTIONS_KEY,
-  ɵSELECTOR_META_KEY
-} from './symbols';
+export * from './symbols';
```

Comment:
We are exposing a lot more here than originally intended.
Do we have an option to not expose so much?

Reply:
Everything which is exported from there is used outside.